server/src/main/java/com/cloud/template/HypervisorTemplateAdapter.java

```diff
@@ -330,8 +342,8 @@ private void createTemplateWithinZone(Long zId, TemplateProfile profile, VMTempl
                         // Check if zone is disabled
                         if (Grouping.AllocationState.Disabled == zone.getAllocationState()) {
                             s_logger.info("Zone " + zoneId_is +
-                                    " is disabled, so skip downloading template to its image store " + imageStore.getId());
-                            continue;
+                                    " is disabled, so uploading template to management server" + imageStore.getId());
+                            //continue;
```

Comment:
don't like this change:
- space removed at end of string so imagestore id gets glued to the word server.
- string says management server biut then it give not the id of the server but on the store, without explanation.
- // continue ==> code in comment, please remove or uncomment